client/boot/common.js

```diff
@@ -87,6 +87,12 @@ const loggedOutMiddleware = currentUser => {
 				page.redirect( '/devdocs/start' );
 			}
 		} );
+	} else {
+		// Break routing if the user is logged out, and tries to visit /
+		page( '/', () => {
+			window.location.href = '/';
+			return;
```

Comment:
It looks like `return` shouldn't be needed here, or should be after the `page()` call.